README.md

```diff
@@ -185,11 +185,11 @@ After the initial setup, here’s how to manage your daily development tasks.
 3. Start NOMAD GUI
 
    ```bash
-   cd packages/nomad-FAIR/gui
-   yarn
-   yarn start
+   uv run poe gui start
    ```
 
+   Tip: `uv run poe gui` maps to `yarn run`, so here you can replace `start` with commands like `test`, `build`, etc.
```

Comment:
I suggest using the GitHub tip box here. 
```suggestion
   > [!TIP]
   > `uv run poe gui` maps to `yarn run`, so here you can replace `start` with commands like `test`, `build`, etc.
```

Reply:
Done